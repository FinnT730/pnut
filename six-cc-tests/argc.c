/* args: abc def hij */
int main(int argc, char_ptr argv) {
  int i = 0;
  printf("Number of arguments: %d\n", argc);
  printf("Arguments:\n");
  while (i < argc) {
    printf("argv[%d] = %s\n", i, argv[i]);
    i++;
  }
}
