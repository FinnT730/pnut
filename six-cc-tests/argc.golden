Number of arguments: 3
Arguments: abc def hij 
